csfieldguide/gulpfile.js

```diff
@@ -0,0 +1,201 @@
+// gulp build : for a one off development build
+// gulp build --production : for a minified production build
+
+'use strict';
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var del = require('del');
+var gulpif = require('gulp-if');
+var exec = require('child_process').exec;
+var runSequence = require('run-sequence')
+var notify = require('gulp-notify');
+var buffer = require('vinyl-buffer');
+var argv = require('yargs').argv;
+
+// sass
+var sass = require('gulp-sass');
+var postcss = require('gulp-postcss');
+var postcssFlexbugFixes = require('postcss-flexbugs-fixes');
+var autoprefixer = require('autoprefixer');
+var sourcemaps = require('gulp-sourcemaps');
+
+// linting
+var jshint = require('gulp-jshint');
+var stylish = require('jshint-stylish');
+
+// Scratch image rendering
```

Comment:
The Scratch image rendering packages and logic can be removed as it's unused in CSFG.